source/overview/security.rst

```diff
@@ -79,40 +138,45 @@ Security updates address newly discovered attacks reported to Mattermost, Inc. b
-   - A summary of security updates made based on past and on-going security analysis and penetration testing.
+A summary of security updates made based on past and on-going security analysis and penetration testing.
 
 - `Mattermost Responsible Disclosure Policy <https://www.mattermost.org/responsible-disclosure-policy/>`_
-   - An overview of how security issues are confidentially reported to and address by Mattermost, Inc.
+An overview of how security issues are confidentially reported to and address by Mattermost, Inc.
 
 Security Policies
 ------------------------------------
 
-For information on internal security policies, development guidelines, business continuity plans and common security-related questions from enterprises, please see our `Security Policies <https://docs.mattermost.com/process/security.html>`_ documentation.
+For information on internal security policies, 
+development guidelines, 
+business continuity plans,
+and common security-related questions from enterprises, 
+please see our `Security Policies <https://docs.mattermost.com/process/security.html>`_ documentation.
 
 
 HIPAA compliance
 ------------------------------------
 
 Deploying Mattermost as part of a HIPAA-compliant IT infrastructure requires a deployment team trained on `HIPAA-compliance requirements and standards <http://www.hhs.gov/hipaa/for-professionals/security/laws-regulations/>`_.
 
-- Mattermost offers HIPAA-relevant **Technincal Safeguards** including:
+Mattermost offers HIPAA-relevant *Technincal Safeguards* including:
 
-     - `Integrity Controls <https://docs.mattermost.com/overview/security.html#integrity-audit-controls>`_
-     - `Access Control <https://docs.mattermost.com/overview/security.html#access-control-policy>`_
-     - `Transmission Security <https://docs.mattermost.com/overview/security.html#transmission-security>`_
-     - `Audit Controls <https://docs.mattermost.com/overview/security.html#integrity-audit-controls>`_
+- Integrity Controls <https://docs.mattermost.com/overview/security.html#integrity-audit-controls>`_
+- Access Control <https://docs.mattermost.com/overview/security.html#access-control-policy>`_
+- Transmission Security <https://docs.mattermost.com/overview/security.html#transmission-security>`_
+- Audit Controls <https://docs.mattermost.com/overview/security.html#integrity-audit-controls>`_.
 
-- HIPAA-compliant deployments commonly consider the following:
+HIPAA-compliant deployments commonly omit the contents of messages from mobile push and email notifications:
 
-     - Omitting the contents of messages from mobile push and email notifications:
+- If your `Push Notifications Contents <https://docs.mattermost.com/administration/config-settings.html#push-notification-contents>`_ option is set to ``Send full message snippet`` there is a chance Personal Health Information (PHI) contained in messages could be displayed on a user's locked phone as a notification. 
+To avoid this, 
```

Comment:
0/5, do we want this as a separate line? 

<img width="763" alt="screenshot 2017-11-15 19 36 04" src="https://user-images.githubusercontent.com/9354841/32867634-4277d850-ca3c-11e7-989a-a7999b3bd775.png">

Reply:
I don't think so....